app/models/scheme.rb

```diff
@@ -187,37 +187,38 @@ def id_to_display
   def check_details_attributes
```

Comment:
do we always use any of these `check_..._attributes` methods together? can we not combine any of them any further?

Reply:
Yup! Refactored it